packages/web-twig/src/Resources/components/TextFieldBase/TextFieldBase.twig

```diff
@@ -0,0 +1,73 @@
+{# API #}
```

Comment:
There is a missing `{%- set props = props | default([]) -%}` which was fixed in https://github.com/lmc-eu/spirit-design-system/pull/499